core/src/com/agateau/pixelwheels/gameinput/DigitalSteering.java

```diff
@@ -19,18 +19,27 @@
 package com.agateau.pixelwheels.gameinput;
 
 import com.agateau.pixelwheels.GamePlay;
-import com.agateau.pixelwheels.racescreen.HudButton;
 import com.badlogic.gdx.math.MathUtils;
 
-class TouchInputUtils {
-    static float applyDirectionInput(HudButton leftButton, HudButton rightButton, float direction) {
-        if (leftButton.isPressed()) {
-            direction += GamePlay.instance.steeringStep;
-        } else if (rightButton.isPressed()) {
-            direction -= GamePlay.instance.steeringStep;
-        } else {
-            direction = 0;
-        }
-        return MathUtils.clamp(direction, -1, 1);
+public class DigitalSteering {
```

Comment:
(This comment is for line 2 but GitHub won't let me comment on it)

You should change the copyright year and owner of this file.

Reply:
done.